server/src/main/java/com/google/coffeehouse/servlets/CreateClubServlet.java

```diff
@@ -104,7 +109,9 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.sendError(HttpServletResponse.SC_BAD_REQUEST, BODY_ERROR);
       return;
     }
-
+    
+    newClub.getCurrentBook().setStorageHandler(handler);
```

Comment:
Would it make sense to set the storageHandler for the book inside Club.save? What is the impact if this line is removed?

Reply:
It would make sense to set the book storage handler inside of club.save(), I cannot imagine a time where we'd want an invocation of club.save() to save the book and the club in separate databases. I've pushed this change.